newjersey/navigator.business.nj.gov
# feat: [#186003172] registration duplicate email, inline error

## Description

Cary wanted inline errors for when your trying to register but your email is already registered 



### Ticket



[Replace with ticket_id](https://www.pivotaltracker.com/story/show/)

### Approach



### Steps to Test

<img width="703" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/89d08235-7795-46f5-925e-efc1910575c7">

Then go to the registartion page and enter an already taken name and hit submit

Should see something like this
<img width="1068" alt="image" src="https://github.com/newjersey/navigator.business.nj.gov/assets/22485301/82a445b3-66ac-4ceb-b165-d75268feb6a1">




### Notes



## Code author checklist
